src/qibolab/channel/channel_map.py

```diff
@@ -2,22 +2,9 @@
 
 from qibo.config import raise_error
 
-from .channel_configs import ChannelConfig
+from . import Channel
```

Comment:
Never import from `__init__.py` within the same subpackage, it will almost certainly cause a circular import (as in this case)
https://github.com/qiboteam/qibolab/blob/2e0ecd658ae9bc216c69f0d494da8e644a9fdcc7/src/qibolab/channel/__init__.py#L4

If you need `Channel` here, define it in this module (changing its name), or in a further one.